pyproject.toml

```diff
@@ -54,7 +54,7 @@ orjson = "^3.8.6"
 SQLAlchemy = ">= 1.4.46, < 2.0"  # note: 1.4.x currently required for enterprise
 mergedeep = "^1.3.4"
 importlib-metadata = "^7.0.1"
-xsdata = {extras = ["cli", "lxml", "soap"], version = ">=22.12,<25.0"}
+xsdata = {extras = ["cli", "lxml", "soap"], version = "=22.12"}
```

Comment:
Dev note: we might have to unpin this eventually to generate the models at a later date, but this pin keeps the diff less noisy until we want to tackle this issue